src/app/components/Images/Images.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styles from './Images.module.css';
+
+export type ImagesProps = {
+  image: 'Mino1' | 'Mino2' | 'Mino3' | 'Mino4';
+};
+export default function Images({ image }: ImagesProps): JSX.Element {
+  return (
+    <img src={`src/assets/MockImages/${image}.png`} className={styles.Images} />
```

Comment:
You need to include your static pictures for storybook, ask Aron how to do that :)
or this https://storybook.js.org/docs/react/api/cli-options#build-storybook

Also this component should accept any picture, not just Mino's. Aim for a prop called `imgSrc` :)